offixsdk/src/main/java/org/aerogear/offixsdk/persistence/MutationDao.kt

```diff
@@ -0,0 +1,27 @@
+package org.aerogear.offixsdk.persistence
+
+import android.arch.persistence.room.*
+import org.aerogear.offixsdk.persistence.Mutation
+
+@Dao
+interface MutationDao {
```

Comment:
I know that you like room but this library seems to be very application level. 
Do you think that it will be a lots of work to use plain SQL client like apollo does in SQL helpers.

Reply:
Is there any specific problem with room? In SQL, have to write a lot of code to achieve this :p

Reply:
Ok. Agreed. Ignore me then. Room is available in core so we do not bring extra bundle into lib